### PR TITLE
Fix #4996

### LINKED ISSUE #4996: [git history]When git-history tab is current tab, click View-Git History can't close git-history tab
### Description
When git-history tab is current tab, click View-Git History can't close git-history tab, while Git，Search and other tab can close.

**Diagnostics:** 

git-history

### PR DESCRIPTION
**Pull request by zixuemeng**
git-history

### CODE REVIEW

**Comment by akosyakov** [comment]
It breaks opening `Git History` for different files:
- open several editors
- trigger `Git History` for one editor via the command palette
- then switch to another editor and try to trigger again

Expected that history gets replaced by content for the second editor, but it gets closed.

**Comment by zixuemeng** [comment]
Like git view,  instead 'Git History' command by 'Toggle Git History View' command.

**Comment by vince-fugnitto** [comment]
@zixuemeng 
You can most likely enable the UI test for the `Git History` since toggling is now enabled.
https://github.com/theia-ide/theia/blob/dcf570d71c328d59988b117f47a0834a3ab49b0e/examples/browser/test/top-panel/top-panel.ui-spec.ts#L153-L163

File: packages/git/src/browser/history/git-history-contribution.ts [hunk 1]
```diff
@@ -35,9 +35,8 @@ export namespace GitHistoryCommands {
     export const OPEN_FILE_HISTORY: Command = {
         id: 'git-history:open-file-history',
     };
-    export const OPEN_BRANCH_HISTORY: Command = {
-        id: 'git-history:open-branch-history',
-        label: GIT_HISTORY_LABEL
+    export const TOGGLE_BRANCH_HISTORY: Command = {
```
Comment (vince-fugnitto) [inline packages/git/src/browser/history/git-history-contribution.ts]: Can we keep the old name?
There may be instances of external extensions calling `executeCommand` with the old const.
This means that those callers will be broken.

Comment (zixuemeng) [reply packages/git/src/browser/history/git-history-contribution.ts]: Ok, I will modify this tomorrow.

Comment (vince-fugnitto) [reply packages/git/src/browser/history/git-history-contribution.ts]: Why do you suggest we have two commands?

Comment (zixuemeng) [reply packages/git/src/browser/history/git-history-contribution.ts]: Is there a command to open branch history when the content of git history is file history?

Comment (vince-fugnitto) [reply packages/git/src/browser/history/git-history-contribution.ts]: From what I understand, the command `Git History` displays the history for a given branch.
If you want the `Git History` for a file, the context menu is used (example: from the explorer, right click a file, and select `Git History`). 

Is there more that should be added?

Comment (zixuemeng) [reply packages/git/src/browser/history/git-history-contribution.ts]: Ok, thank you for your explanation, I understand. I will update this commit to keep the old const name and enable the UI test for git history tomorrow.

Comment (zixuemeng) [reply packages/git/src/browser/history/git-history-contribution.ts]: done.

**Pull request merged**